### ISSUE TITLE
Support lesson collections in multiple languages (Oppgavesamlinger)

### ISSUE THREAD

**Issue by krskog** [kodeklubben/codeclub-viewer#271]
Support lesson collections in multiple languages.

Lessons are hardcoded in [.txt files](https://github.com/kodeklubben/oppgaver/tree/master/src/scratch/playlists) and displayed with the name of the .txt file. Possible solutions could be to do it the same way as [captions](https://github.com/kodeklubben/codeclub-viewer/tree/master/src/constants), or have .txt files for each language with tags.

**Comment by Skagevang** [comment]
Locally lesson collections don't work

**Comment by NorwegianKiwi** [comment]
Suggestion: Create playlists as yaml-files inside the `playlists`-folder (next to the txt-files; keep these for now so that we don't break the old pages) that list the folders that the lessons are in (thus making the playlists language-independent). The only language-dependent with them is the title, so list those. Also, add a sort-key to indicate the order they should be shown. F.ex.:
```
sortindex: 1
title:
    en: "My playlist"
    nb: "Min spilleliste"
lessons: [astrokatt, straffespark, gangemesteren]
```
The `sortindex` controls the order of the playlists relative to each other. Smaller numbers come first.

The name of the playlist-files is not important.

**Comment by NorwegianKiwi** [comment]
Having playlists in several languages, means that we might have playlists where not all the lessons have been translated to your current language. Suggestion for how to deal with this: If there is at least one lesson that is missing from a playlist (because it has not been translated), only show the title of the playlist and a message saying "The lessons in this collection have not been translated to <your language> yet."

**Comment by Skagevang** [comment]
Could it also be a message encouraging the user to translate those that's missing? Or is this too much and it could create "chaos" in the lesson repo?

**Comment by NorwegianKiwi** [comment]
Fixed in #506
